tests/testthat/test-categories.R

```diff
@@ -318,6 +318,44 @@ with_test_authentication({
             expect_equivalent(as.array(crtabs(~ v4e, data=ds)),
                 array(c(10, 10), dim=2L, dimnames=list(v4=c("B", "D"))))
         })
+
+        test_that("Can changeCategoryID", {
+            ds$v4f <- df$v4
+            expect_identical(names(categories(ds$v4f)),
+                c("B", "C", "No Data"))
+            expect_equal(ids(categories(ds$v4f)),
+                c(1, 2, -1))
+            ds$v4f <- changeCategoryID(ds$v4f, 2, 6)
+            expect_identical(names(categories(ds$v4f)),
+                c("B", "C", "No Data"))
+            expect_equal(ids(categories(ds$v4f)),
+                c(1, 6, -1))
+            expect_error(ds$v4f <- changeCategoryID(ds$v4f, 2, 7),
```

Comment:
Move these input validations to unit tests that run in the mock context above, similar to this pattern: https://github.com/Crunch-io/rcrunch/blob/master/tests/testthat/test-join.R#L44-L63

That way, we get some unit test coverage on this function and don't have as steep of a coverage drop.